src/Subscriber/ConfigurationSubscriber.php

```diff
@@ -68,6 +72,19 @@ public function onPageLoaded(ShopwareSalesChannelEvent $event): void
         }
     }
 
+    private function getPage(ShopwareSalesChannelEvent $event)
+    {
+        if (method_exists($event, 'getPage')) {
+            return $event->getPage();
+        }
+
+        if (isset($event->getParameters()['page'])) {
+            return $event->getParameters()['page'];
+        }
+
+        throw new \Exception(sprintf('Unable to get page from event %s.', get_class($event)));
```

Comment:
it is better to add `use Exception` directive and use it without `\` inside the code

Reply:
fixed